x11-packages/mpv-x/default-to-x11.patch

```diff
@@ -0,0 +1,27 @@
+By default mpv chooses wrong video backend
+```
+Error opening/initializing the selected video_out (--vo) device.
+Video: no video
+```
+This patch forces it to choose X11 by default.
++++ ./video/out/vo.c
+@@ -69,6 +69,9 @@
+ 
+ static const struct vo_driver *const video_out_drivers[] =
+ {
++#if HAVE_X11
++    &video_out_x11,
++#endif
```

Comment:
How does this change anything?
It just moves the `HAVE_X11` check up in the struct declaration.
Does the order determine what the default is?

Reply:
https://github.com/mpv-player/mpv/blob/37159a8ee03d155e4b139a5bf0a50bee910cad69/video/out/vo.c#L70C38-L117
https://github.com/mpv-player/mpv/blob/37159a8ee03d155e4b139a5bf0a50bee910cad69/video/out/vo.c#L350-L357
It iterates through all drivers and checks everyone for being loaded.
As far as I understand it does not move further after android's mediacodec backend and prints error.

Reply:
So order matters.

Reply:
Maybe we should move the Android one to the end of the list instead then?

Reply:
We can remove it completely since it uses jni and in our case it does not make sense.

Reply:
Probably makes more sense.